docs/CHANGELOG.rst

```diff
@@ -15,6 +15,15 @@ Changed
 - **BACKWARD INCOMPATIBLE:** Require Resolwe 17.x
 - **BACKWARD INCOMPATIBLE:** Use Elasticsearch version 6.x
 - **BACKWARD INCOMPATIBLE:** Bump Django requirement to version 2.2
+- **BACKWARD INCOMPATIBLE:** Remove obsolete RNA-seq workflows
+  ``workflow-bbduk-star-featurecounts-single``,
+  ``workflow-bbduk-star-featurecounts-paired``,
+  ``workflow-cutadapt-star-featurecounts-single`` and
```

Comment:
The workflows `workflow-cutadapt-star-featurecounts-single` and `workflow-cutadapt-star-featurecounts-paired` are removed in one commit but mentioned in the change log in another commit.